build.gradle

```diff
@@ -25,7 +24,7 @@ buildscript {
 
   ext.deps = [
       'picocli': "info.picocli:picocli:${versions.picocli}",
-      'android': "com.google.android:android:${versions.android}",
+      'android': "org.robolectric:android-all:9-robolectric-4913185-2",
```

Comment:
that’s annoying that Google stopped publishing their APIs to Maven Central.

They aren’t on the Google repo by chance?

Reply:
Google never did. Myself and a few others were responsible for these by building AOSP ourselves and validating binary compatibility.

Reply:
Any objection to android-all here?

Reply:
None from me.